submissions/innazozulia/DOM practice/main.js

```diff
@@ -0,0 +1,153 @@
+    description: 'Charmeleon mercilessly destroys its foes using its sharp claws. If it encounters a strong foe, it turns aggressive. In this excited state, the flame at the tip of its tail flares with a bluish white color.'
+  },
+  {
+    name: 'Squirtle',
+    imgSrc: './img/Squirtle.jpeg',
+    description: 'Squirtles shell is not merely used for protection. The shells rounded shape and the grooves on its surface help minimize resistance in water, enabling this Pokémon to swim at high speeds.'
+  },
+  {
+    name: 'Wartortle',
+    imgSrc: './img/Wartortle.jpeg',
+    description: 'Its tail is large and covered with a rich, thick fur. The tail becomes increasingly deeper in color as Wartortle ages. The scratches on its shell are evidence of this Pokémons toughness as a battler.'
+  },
+  {
+   name: 'Caterpie',
+   imgSrc: './img/Caterpie.jpeg',
+   description: 'Caterpie has a voracious appetite. It can devour leaves bigger than its body right before your eyes. From its antenna, this Pokémon releases a terrifically strong odor.'
+  },
+  {
+    name: 'Metapod',
+    imgSrc: './img/Metapod.jpeg',
+    description: 'The shell covering this Pokémons body is as hard as an iron slab. Metapod does not move very much. It stays still because it is preparing its soft innards for evolution inside the hard shell.'
+  },
+  {
+    name: 'Butterfree',
+    imgSrc: './img/Butterfree.jpeg',
+    description: 'Butterfree has a superior ability to search for delicious honey from flowers. It can even search out, extract, and carry honey from flowers that are blooming over six miles from its nest.'
+  },
+  {
+    name: 'Weedle',
+    imgSrc: './img/Weedle.jpeg',
+    description: 'Weedle has an extremely acute sense of smell. It is capable of distinguishing its favorite kinds of leaves from those it dislikes just by sniffing with its big red proboscis (nose).'
+  },
+  {
+    name: 'Kakuna',
+    imgSrc: './img/Beedrill.jpeg',
+    description: 'Kakuna remains virtually immobile as it clings to a tree. However, on the inside, it is extremely busy as it prepares for its coming evolution. This is evident from how hot the shell becomes to the touch.'
+  },
+  {
+    name: 'Beedrill',
+    imgSrc: './img/Kakuna.jpeg',
+    description: 'Beedrill is extremely territorial. No one should ever approach its nest—this is for their own safety. If angered, they will attack in a furious swarm.'
+  },
+  {
+    name: 'Pidgey',
+    imgSrc: './img/Pidgey.jpeg',
+    description: 'Pidgey has an extremely sharp sense of direction. It is capable of unerringly returning home to its nest, however far it may be removed from its familiar surroundings.'
+  },
+  {
+    name: 'Rattata',
+    imgSrc: './img/Rattata.jpeg',
+    description: 'Rattata is cautious in the extreme. Even while it is asleep, it constantly listens by moving its ears around. It is not picky about where it lives—it will make its nest anywhere.'
+  },
+];
+
+const container = Document.createElement('div');
```

Comment:
You can create this block and other static elements in your html file. It can make your js code cleaner a little bit